src/Microsoft.ApplicationInsights.AspNetCore/ApplicationInsightsJavaScript.cs

```diff
@@ -59,7 +64,7 @@ public string FullScript
                         identity != null &&
                         identity.IsAuthenticated)
                     {
-                        string escapedUserName = identity.Name.Replace("\\", "\\\\");
+                        string escapedUserName = encoder.Encode(identity.Name);
```

Comment:
It was from this PR though: https://github.com/Microsoft/ApplicationInsights-aspnetcore/pull/411